css/alert_banner.css

```
@@ -158,13 +161,13 @@
   -ms-transform: rotate(45deg);
   -o-transform: rotate(45deg);
   filter: progid:DXImageTransform.Microsoft.BasicImage(rotation=0.5);
-  line-height: 1;
+  line-height: 0;
 }
-.govcms-alerts article.node--type-alert .govcms-alert-message time {
+.alert-banners article.node--type-alert .alert-message time {
```

```suggestion
.alert-banners article.node--type-alert-banner .alert-banner-message time {
```
@govindmaloo `govcms-alert` must be changed to `alert-banner`.

---

@sonnykt Updated